src/pages/WorkspaceDetails/index.tsx

```diff
@@ -153,6 +154,7 @@ export class WorkspaceDetails extends React.PureComponent<Props, State> {
 
     return (
       <React.Fragment>
+        <Head pageName="Workspace Details" />
```

Comment:
As we currently do, it's better to use workspace name here.

Reply:
I have fixed it.